packages/powersync/lib/src/open_factory.dart

```diff
@@ -136,3 +106,33 @@ class PowerSyncOpenFactory extends DefaultSqliteOpenFactory {
     );
   }
 }
+
+extension on Abi {
+  String get localName {
```

Comment:
This feels weird to have as an extension on `Abi`, since it's just an internal function used by powersync. `Abi.current().localName` doesn't give any indication that it's a powersync-related function. It also causes two calls to `Abi.current()`.

Can we make it a plain function instead?

Reply:
Updated it to a plain function that returns the path and we can override it for tests